### ISSUE TITLE
Specify sassLoadPaths example

### ISSUE THREAD

**Issue by scottrogowski** [xzyfer/gulp-sass-graph#7]
I'm having trouble getting this plugin to work. Could you include an example of exactly what goes into sassLoadPaths?

**Comment by misteroh** [comment]
i dont know if he's maintaining this repo anymore? i'm having a hell of a time trying to get this to work as well

**Comment by zobzn** [comment]
example:
var sassLoadPaths = ['./dir1', './dir2'];
...
.pipe(sassGraph(sassLoadPaths))
.pipe(sass({loadPath: sassLoadPaths}))
...

**Comment by fokkerone** [comment]
doesn´t work!

**Comment by eranimo** [comment]
libsass doesn't even have a loadPath attribute

**Comment by zobzn** [comment]
ups...
this must work
var sassLoadPaths = ['./dir1', './dir2'];
...
.pipe(gulp_sass_graph(sassLoadPaths))
.pipe(gulp_sass({includePaths: sassLoadPaths})

**Comment by towry** [comment]
。。。。。

Totally not working.

**Comment by ViieeS** [comment]
@towry yes, it is. Use [gulp-sass-inheritance](https://www.npmjs.com/package/gulp-sass-inheritance) instead.
